src/js/components/stock-movement-wizard/outbound/CreateStockMovement.jsx

```diff
@@ -242,6 +242,23 @@ class CreateStockMovement extends Component {
       .catch(() => this.props.hideSpinner());
   }
 
+  fetchDestinations() {
+    const url = '/openboxes/api/locations?name=&direction=OUTBOUND';
+    return apiClient.get(url)
```

Comment:
This pulls all locations (around 1200) from the backend. I think, that when we were talking about this feature, we wanted just to show a few example locations for users to know what to type there and if the user types something in it will be pulled from the backend. Unless the requirements have changed since the tech huddle we had with @jmiranda.